### PR TITLE
Fix coord2area_def.py utillity script for python 3

### PR DESCRIPTION
**Pull request by adybbroe**
Fixing the utility script used to help users define a new area. Now it should be Python 3 compatible and it outputs the "new" area definition format for the areas.yaml file.

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Tests passed 
 - [x] Passes ``git diff origin/master -- "*py" | flake8 --diff`` 
 - [ ] Fully documented

### CODE REVIEW

File: utils/coord2area_def.py [hunk 1]
```diff
@@ -104,19 +104,33 @@
...
+    print("\tNAME:\t", name)
+    print("\tPCS_ID:\t", proj + "_" + str(lon_0) + "_" + str(lat_0))
+    print("\tPCS_DEF:\tproj=" + proj +
+          ",lat_0=" + str(lat_0) +
+          ",lon_0=" + str(lon_0) +
+          ",ellps=WGS84")
+    print("\tXSIZE:\t", xsize)
+    print("\tYSIZE:\t", ysize)
+    print("\tAREA_EXTENT:\t", area_extent)
+    print("};")
```
Comment (djhoese) [inline utils/coord2area_def.py]: I personally think we should just get rid of the old format.

Comment (adybbroe) [reply utils/coord2area_def.py]: @djhoese Agreed!

**Comment by adybbroe** [comment]
Are we okay with this as it is now? @djhoese and @mraspaud ?

**Comment by djhoese** [comment]
@adybbroe Have you tested that the YAML produced can be read by pyresample?

**Comment by adybbroe** [comment]
> @adybbroe Have you tested that the YAML produced can be read by pyresample?

@djhoese Yes! I took the output and pasted in the yaml file and it works. Enough?

**Pull request merged**